src/realm.h

```diff
@@ -2546,6 +2546,8 @@ RLM_API void realm_app_config_set_default_request_timeout(realm_app_config_t*, u
 RLM_API void realm_app_config_set_platform(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
 RLM_API void realm_app_config_set_platform_version(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
 RLM_API void realm_app_config_set_sdk_version(realm_app_config_t*, const char*) RLM_API_NOEXCEPT;
+RLM_API void realm_app_config_set_in_memory(realm_config_t*, uint8_t) RLM_API_NOEXCEPT;
```

Comment:
The names of the functions are wrong. This is not a property on the app config, but on the realm config. The functions should be placed together with the other functions in that category.

Reply:
Thanks @jedelbo . Moved everything into the right place now and added 2 small tests..